### PR TITLE
Allow setting vertical line spacing with lineSpacing property in GUI Textblock control

### PR DESCRIPTION
**Pull request by CarlosLanderas**
When working with multiple lines, textblock is using current line measure for vertical spacing, but lines are very close as you can see in the following image:

I added in textblock the property lineSpacing so you can apply vertical spacing between lines like this:

```javascript
 var text1 = new BABYLON.GUI.TextBlock();
 text1.textWrapping = true;
 text1.lineSpacing = "80px";
```

Or using percentage:

```javascript
 text1.lineSpacing = "20%";
```

Textblock output:

### CODE REVIEW

File: gui/src/controls/textBlock.ts [hunk 1]
```diff
@@ -20,6 +20,10 @@ module BABYLON.GUI {
             return this._resizeToFit;
         }
 
+        public set lineSpacing(value: number) {
```
Comment (RaananW) [inline gui/src/controls/textBlock.ts]: Not sure if this is the convention in the GUI, but why is there only a setter and no getter?

Comment (CarlosLanderas) [reply gui/src/controls/textBlock.ts]: Added the getter :).

File: gui/src/controls/textBlock.ts [hunk 2]
```diff
@@ -81,6 +81,14 @@ module BABYLON.GUI {
             this._markAsDirty();
         }
 
+        public set lineSpacing(value: number) {
+            this._lineSpacing = value;
+        }
+
+        public get lineSpacing(): number {
+            return this._lineSpacing;
```
Comment (deltakosh) [inline gui/src/controls/textBlock.ts]: You need to call markAsDirty to force render. But make sure the value actually changed

Comment (deltakosh) [reply gui/src/controls/textBlock.ts]: Also this has to be an UnitOrValue object to allow users to use pixel or percentage

Comment (CarlosLanderas) [reply gui/src/controls/textBlock.ts]: Cool @deltakosh, thanks for the tips :D.

**Comment by deltakosh** [comment]
Great. Please also update documentation and credits yourself in the what's new:
- https://github.com/BabylonJS/Documentation/blob/master/content/How_To/GUI/Gui.md
- https://github.com/BabylonJS/Babylon.js/blob/master/dist/preview%20release/what's%20new.md

**Comment by CarlosLanderas** [comment]
Will do ASAP. Thanks.

**Pull request merged**